### DISCUSSION TITLE
How do i show "remote images" from TMDB api?

### DISCUSSION THREAD

**Discussion by bappyasif** [vercel/next.js#46880] [Help] [answered]
### Summary

hello everybody, currently browser shows me "invalid url" (1*) and asks me to configure "host name" (2 *), which i did as per documentation from [nextJs](https://nextjs.org/docs/messages/next-image-unconfigured-host)

here is what my config settings looks like : 
```js
images: {
    remotePatterns: [
      {
        protocol: 'https',
        hostname: 'image.tmdb.org',
        port: '',
        pathname: 'https://image.tmdb.org/t/p/original',
      },
    ],
  },
  ```
  
in rendering function component im using it as such:
1* > `<Image src={poster_path} width={330} height={220} />`  // this shows invalid url
i even tried  this:
2* > `<Image src={`https://image.tmdb.org/t/p/original/130H1gap9lFfiTF9iDrqNIkFvC9.jpg`} width={330} height={220} />`
and getting this error
![image](https://user-images.githubusercontent.com/19854521/223459853-6b284985-3313-47d9-9597-18cd3736ef97.png)

what do i have to show these "remote" images from TMDB to show up on DOM, thanks for looking into this, much appreciated :)

### Additional information

```js
https://github.com/vercel/next.js/discussions/38281

i even tried to restart my dev server to see if that helps or not, but still same error on browser!!
```

**Comment by bappyasif** [comment]
i also updated config file after giving nextjs docs another read for "remote images"

```js
images: {
    remotePatterns: [
      {
        protocol: 'https',
        hostname: 'image.tmdb.org',
        port: '',
        pathname: '/t/p/original',
      },
    ],
  },
  ```
  and in rendering with Image element as such:
  `<Image src={`${poster_path}`} width={330} height={220} />`
  
  but i can see its showing me "broken url" and not attaching these "remote patterns" in that at all.... but getting this 400 response with an "invalid img src url" fetch request
![image](https://user-images.githubusercontent.com/19854521/223465794-1e2ee761-8d15-4aea-ace7-bec26772d32d.png)

**Comment by wwwdepot** [comment, marked as answer]
Error doc does answer the problem, but you left out the wildcard symbol `**` in your config.


Read https://nextjs.org/docs/api-reference/next/image#remote-patterns

I say if you don't know what's going on, you can leave those pathname out & use hostname only or use [`Domains`](https://nextjs.org/docs/api-reference/next/image#domains).

```js
    remotePatterns: [
      { hostname: "images.unsplash.com" },
      { hostname: "miro.medium.com" },
      {
        protocol: "https",
        hostname: "image.tmdb.org",
        pathname: "/t/p/original/**",
      },
    ],
```

**Reply by bappyasif** [reply]
thanks a lot, it makes a lot more sense to me now :)

didnt realize that was a "wildcard", noob mistake, thanks for helping me out out, much appreciated :)
